plugin/pkg/shoot/validator/admission.go

```diff
@@ -391,6 +393,13 @@ func (c *validationContext) validateScheduling(ctx context.Context, a admission.
 				return admission.NewForbidden(a, fmt.Errorf("cannot schedule shoot '%s' on seed '%s' that already has the maximum number of shoots scheduled on it (%d)", c.shoot.Name, c.seed.Name, allocatableShoots.Value()))
 			}
 		}
+
+		if schedulerfeatures.FeatureGate.Enabled(features.HAControlPlanes) {
```

Comment:
I think you should use:
```suggestion
		if utilfeature.DefaultFeatureGate.Enabled(features.HAControlPlanes) {
```
Where `utilfeatures` is `utilfeature "k8s.io/apiserver/pkg/util/feature"`

Reply:
This has now been removed completely.